lib/binance/futures.ex

```diff
@@ -147,4 +147,235 @@ defmodule Binance.Futures do
+          t
+      end
+
+    arguments =
+      %{
+        symbol: symbol,
+        side: side,
+        type: type,
+        quantity: quantity,
+        timestamp: timestamp,
+        recvWindow: receiving_window
+      }
+      |> Map.merge(
+        unless(
+          is_nil(new_client_order_id),
+          do: %{newClientOrderId: new_client_order_id},
+          else: %{}
+        )
+      )
+      |> Map.merge(unless(is_nil(stop_price), do: %{stopPrice: stop_price}, else: %{}))
+      |> Map.merge(unless(is_nil(time_in_force), do: %{timeInForce: time_in_force}, else: %{}))
+      |> Map.merge(unless(is_nil(price), do: %{price: price}, else: %{}))
+
+    case HTTPClient.post_binance("/fapi/v1/order", arguments) do
+      {:ok, %{"code" => code, "msg" => msg}} ->
+        {:error, {:binance_error, %{code: code, msg: msg}}}
+
+      data ->
+        data
+    end
+  end
+
+  @doc """
+  Creates a new **limit** **buy** order
+
+  Symbol can be a binance symbol in the form of `"BTCUSDT"`
+
+  Returns `{:ok, %{}}` or `{:error, reason}`
+  """
+  def order_limit_buy(symbol, quantity, price, time_in_force \\ "GTC")
+      when is_binary(symbol)
+      when is_number(quantity)
+      when is_number(price) do
+    create_order(symbol, "BUY", "LIMIT", quantity, price, time_in_force)
+    |> parse_order_response
+  end
+
+  @doc """
+  Creates a new **limit** **sell** order
+
+  Symbol can be a binance symbol in the form of `"BTCUSDT"` or `%Binance.TradePair{}`.
+
+  Returns `{:ok, %{}}` or `{:error, reason}`
+  """
+  def order_limit_sell(symbol, quantity, price, time_in_force \\ "GTC")
+      when is_binary(symbol)
+      when is_number(quantity)
+      when is_number(price) do
+    create_order(symbol, "SELL", "LIMIT", quantity, price, time_in_force)
+    |> parse_order_response
+  end
+
+  @doc """
+  Get all open orders, alternatively open orders by symbol
+
+  Returns `{:ok, [%Binance.Futures.Order{}]}` or `{:error, reason}`.
+
+  Weight: 1 for a single symbol; 40 when the symbol parameter is omitted
+
+  ## Example
+  ```
+  {:ok,
+    [%Binance.Futures.Order{price: "0.1", orig_qty: "1.0", executed_qty: "0.0", ...},
+     %Binance.Futures.Order{...},
+     %Binance.Futures.Order{...},
+     %Binance.Futures.Order{...},
+     %Binance.Futures.Order{...},
+     %Binance.Futures.Order{...},
+     ...]}
+  ```
+
+  Read more: https://binanceapitest.github.io/Binance-Futures-API-doc/trade_and_account/#current-open-orders-user_data
+  """
+  def get_open_orders() do
+    api_key = Application.get_env(:binance, :api_key)
+    secret_key = Application.get_env(:binance, :secret_key)
+
+    case HTTPClient.get_binance("/fapi/v1/openOrders", %{}, secret_key, api_key) do
+      {:ok, data} -> {:ok, Enum.map(data, &Binance.Futures.Order.new(&1))}
+      err -> err
+    end
+  end
+
+  def get_open_orders(symbol) when is_binary(symbol) do
+    api_key = Application.get_env(:binance, :api_key)
+    secret_key = Application.get_env(:binance, :secret_key)
```

Comment:
@trexnix can you move these info `api_key = Application.get_env(:binance, :api_key)` into attribute variables ?

Reply:
I updated @kimquy